packages/dds-components/src/components/Tabs/TabList.tsx

```diff
@@ -29,27 +30,38 @@ export const TabList = forwardRef<HTMLDivElement, TabListProps>(
       tabListRef,
       hasTabFocus,
       setHasTabFocus,
+      addTabButtonProps,
     } = useTabsContext();
 
     const uniqueId = id ?? `${tabsId}-tablist`;
     const childrenArray = Children.toArray(children).length;
     const [focus, setFocus] = useRoveFocus(childrenArray, hasTabFocus, 'row');
     const combinedRef = useCombinedRef(ref, tabListRef);
 
-    const tabListChildren = Children.map(children, (child, index) => {
-      return (
-        isValidElement(child) &&
-        cloneElement(child as ReactElement, {
-          id: `${tabsId}-tab-${index}`,
-          'aria-controls': `${tabsId}-panel-${index}`,
-          active: activeTab === index,
-          index,
-          focus: focus === index && hasTabFocus,
-          setFocus,
-          onClick: () => handleTabChange(index),
+    const hasButton = addTabButtonProps ? true : false;
+
+    const tabListChildren = Children
+      ? Children.map(children, (child, index) => {
+          return (
+            isValidElement(child) &&
+            cloneElement(child as ReactElement, {
+              id: `${tabsId}-tab-${index}`,
+              'aria-controls': `${tabsId}-panel-${index}`,
+              active: activeTab === index,
+              index,
+              focus: focus === index && hasTabFocus,
+              setFocus,
+              onClick: () => handleTabChange(index),
```

Comment:
Kanskje out-of-scope for denne PR-en, men nå overskriver vi `onClick`, som kan være kjipt hvis noen har lyst til å gjøre noen ekstra handlinger når knappen trykkes på. Burde vi heller kalle begge i de tilfeller den allerede har en `onClick`? 🤔 
```suggestion
              onClick: () => {
                child.props.onClick?.()
                handleTabChange(index)
              },
```

Reply:
Godt poeng. Lager egen oppgave på det 👍